cryptoinvestor/models/user.py

```diff
@@ -57,16 +57,7 @@ def sell(self, count: float, a_name: str) -> float:
 
         s = count * price
 
-        self.bought[a_name] -= count
-        self._sell(s)
+        self.account.investments[a_name]['owned'] -= count
+        self.account.sell(s)
 
         return s
-
-    def _buy(self, sum: float):
-        self.account.balance -= sum
-
-    def _sell(self, sum: float):
-        self.account.balance += sum
-
-    def balance(self) -> float:
```

Comment:
Why removing this? This is just a helper function to get current balance of the user w/o having to access his account.
Leaving this also removes the need for changing tests.

Reply:
Changed test were already committed, should we remove this commit ?

Reply:
Please do